examples/Azure_IoT_Adu_ESP32/readme.md

```diff
@@ -71,7 +71,7 @@ _The following was run on Windows 11 and Ubuntu Desktop 20.04 environments, with
 
     - On the Arduino IDE, go to menu `Sketch`, `Include Library`, `Manage Libraries...`.
     - Search for and install `azure-sdk-for-c`.
-    - TODO: Make sure to install the `<INSERT BETA VERSION>` to use the preview version of the Embedded C SDK.
+    - **Make sure to install the `1.1.0-beta.1` to use the preview version of the Embedded C SDK.**
```

Comment:
I don't see this version, do you mean 1.0.0-beta-1?
![image](https://user-images.githubusercontent.com/54871851/197890830-94d10e60-de5e-4240-8386-07d3aec0665a.png)

Reply:
https://github.com/Azure/azure-sdk-for-c-arduino/releases/tag/1.1.0-beta.1

The release is here and should hopefully get pulled in the next time the Arduino Library manager updates its index (should do it every hour).